fluent-bit/README.md

```diff
@@ -2,6 +2,68 @@
 
 This image is based on the official [Fluent Bit Docker image](https://github.com/fluent/fluent-bit-docker-image).
 
-Changes are made to be based on on the latest debian:testing-slim in contrast to distroless, that is always debian:stable based.
+We add modifications so that we use the latest `debian:testing-slim` variant, as opposed to the upstream distroless image, which is always `debian:stable`-based.
 
-It contains Fluent Bit and the plugin for Loki.
+The custom plugin has the following customizations:
+
+* Loki output plugin
+* Sequential HTTP output plugin
+
+## Sequential HTTP output plugin
+
+The standard HTTP output plugin `out-http` sends out records in batch. This is a problem for some consuming services in the SKR environment (e.g. SAP Audit Service) 
+The `out-sequentialhttp` is a drop-in replacement that sends out records sequentially (one-by-one).
+
+### Code modifications
+
+The code of `out-sequentialhttp` is almost identical to `out-http`. The only difference is how msgpack to JSON transcoding/sending to the HTTP backend is done in the `cb_http_flush` function.  
+
+### Functional testing
+
+To test the `out-sequentialhttp` plugin on an SKR cluster, deploy a mock HTTP server, and make the `sequentialhttp` plugin send the logs to the mock.
+
+1. Deploy a mock server in the `kyma-system` Namespace:
+```bash
+kubectl create -f https://raw.githubusercontent.com/istio/istio/master/samples/httpbin/httpbin.yaml
+```
+
+2. Send dummy logs to the mock server. To do this, edit the `logging-fluent-bit-config` ConfigMap, and replace the following plugins in the `extra.conf` section:
+* `http` output plugin → the `sequentialhttp` plugin
+* `tail` input plugin → the `dummy plugin` to simulate sending audit log at a high rate (otherwise no batching would occur)
+
+An example configuration looks as follows:
```

Comment:
```suggestion
An example plugin configuration looks as follows:
```

I would specify here that this is the plugin. It might be cos I'm not a dev, but for me it wasn't obvious that this is the plugin until I got to step 5.

Reply:
As per discussion, changed to Fluent Bit